includes/push-ui.php

```diff
@@ -23,7 +23,7 @@ function() {
 			add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\enqueue_scripts' );
 			add_filter( 'amp_dev_mode_element_xpaths', __NAMESPACE__ . '\add_element_xpaths' );
 			add_filter( 'script_loader_tag', __NAMESPACE__ . '\add_dev_mode_to_assets', 10, 2 );
-			add_action( 'wp_ajax_dt_load_connections', __NAMESPACE__ . '\get_connections' );
+			add_action( 'wp_ajax_dt_load_connections_push', __NAMESPACE__ . '\get_connections' );
```

Comment:
I'm assuming this is for clarity from `wp_ajax_dt_load_connections_pull`? 

We're probably stuck with the old name to ensure backward compatibility with the third party extensions and between different versions of the plugin. Otherwise we could go through a soft deprecation.